src/utils/common-utils.ts

```diff
@@ -90,32 +80,16 @@ const extensions = [
     }
 ];
 
-export const shuffle = (a: any[]) => {
```

Comment:
Note: unrelated performance improvement to remove unused logic for random sampling